src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -942,6 +941,7 @@ public void testRawGtCountAnnotation() {
                 Assert.assertEquals(rawGtCount.get(0), ".");
```

Comment:
The issue is that `RawGtCount` is combined by GenomicsDB as a sum. There is no `RawGtCount` annotation on RefBlocks so the HomRef count always ends up being 0 in this case. It seems like `ExcessHet` isn't actually using `RawGtCount` as I expected. It recalculates the counts in `GenotypeUtils.computeDiploidGenotypeCounts` which doesn't seem to use `RawGtCount` so I'm not sure why `RawGtCount` is the `rawKey` associated with `ExcessHet`. Seems like this whole thing could be cleaned up so that we use the same method `ExcessHet` does to calculate the count of Het and HomVar genotypes at a site, but I'm not sure why we didn't do that in the first place.

Also I added the documentation to the header line.